include/net/lwm2m.h

```diff
@@ -318,6 +319,76 @@ struct coap_block_context *lwm2m_firmware_get_block_context();
 #endif
 #endif
 
+
+#if defined(CONFIG_LWM2M_SWMGMT_OBJ_SUPPORT)
```

Comment:
Seems wrong to have these prototypes inside the lwm2m header, doesn't seem as if any other objects do it like this?

Reply:
FOTA does this just above... Copied it from there